.storybook/utils.ts

```diff
@@ -79,3 +102,41 @@ export const hideAllStorybookControls = {
 export const hidePanel = {
   options: { showPanel: false },
 };
+
+/**
+ * Internal helper function to merge `argTypes` objects into
+ * a Storybook story config object
+ *
+ * @returns the mutated config
+ */
+const _updateArgTypes = <Props>(
+  config: StorybookConfig<Props>,
+  propNames: Array<keyof Props>,
+  { key, value }: { key: string; value: Record<string, any> | boolean | string }
```

Comment:
not sure if I'm way off here, but this seems more accurate to what our current configurations are?
```suggestion
  { key, value }: { key: string; value: Record<string, boolean | string> | boolean }
```

Reply:
Yes, for the current usage. I just kept it more open with `any` as the [argTypes](https://storybook.js.org/docs/api/arg-types) can be a lot of things actually.

```js
control: false

control: {
  description: string,
  max: number
  labels: { [option: string]: string }
  presetColors: string[]
  ...
}

table: {
  disable: true
}

...
```

I'm ok making it stricter based on current usage and we can open it up if needed in the future (e.g. for more utils). 👍